requirements-cuda.txt

```diff
@@ -4,7 +4,7 @@
 # Dependencies for NVIDIA GPUs
 ray >= 2.9
 nvidia-ml-py # for pynvml package
-torch == 2.4.0
+torch == 2.5.0
```

Comment:
Only concern here is now `torch` uses the `12.4 cuda` bindings by default. We might want to update the installation docs (including on the readme) to alert users that they may want to pass `--extra-index-url https://download.pytorch.org/whl/cu121` during installation

Reply:
@robertgshaw2-neuralmagic Does this mean that there would need to be multiple vllm packages (one for 12.1 and one for 12.4)?  Or should I try to install pytorch 2.5 built with 12.1 (if such a thing exists)?